### DISCUSSION TITLE
2023-11-06

### DISCUSSION THREAD

**Discussion by miniposi** [10-rolling/rollingPolling#29] [Daily_Meeting]
### ✅ 어제 해낸 일

* 컴포넌트 구현

### 📝 회의내용

* theme 구조
```js
import theme from 'styles/theme';
 ${theme.fontWeight.normal};
```
* font-size 변경여부
-> 수님이 하시는 거

* 각자 조사 해온 거 발표
-> 완료

* hook 하나 개발
zustand (통신 X) vs react-query (통신 O)

* 기능 정리해서 나누기
-> style, img, icon 명 변경, button cursor

* 다음 개발 역할 분담

### 💪🏻 오늘 할일

* 각자 styles 수정
* 분담 받은 역할

### 🤔 특이사항

프로젝트 시작하고 바빠서 포시가 삐졌어요
![KakaoTalk_Photo_2023-11-06-13-13-46 001](https://github.com/10-rolling/rollingPolling/assets/52947668/1379e515-af28-4b5c-bd00-e69a5a57d43c)

**Comment by hanseulhee** [comment]
포시야 삐지지마 ~~~
